task/buildah/0.2/buildah.yaml

```diff
@@ -25,7 +26,7 @@ spec:
     description: Reference of the image buildah will produce.
   - name: BUILDER_IMAGE
     description: The location of the buildah builder image.
-    default: quay.io/buildah/stable:v1.17.0
+    default: quay.io/buildah/stable:v1.18.0@sha256:120f0bf114bf02c14db672c5940615db220fb4be7a8a2e73e11fb48ff7f510de
```

Comment:
can you please remove the digest from the image as whenever there is a push on this tag the previous digest becomes in-accessible. This is mainly the scenario in quay and might break the task in future :sweat_smile:

Reply:
Sure, sorry, I didn't know. I just followed hints after catlin check :)